GSA/idmanagement.gov
# 0221-idm-link-fixes

Updated broken links in report from SiteImprove: `February Broken Links Report`

Closes #801 

Link Updates: 
- On Page: https://www.idmanagement.gov/fips201ep/#test-card-loaners	
  - Updated subnav to: #test-card-loaners
  - This fixed broken subnav issue. 
  - Reported by: @SuGhadiali 
- On Page: https://www.idmanagement.gov/vendors/
  - Fixed link in section: Step 3 – Get on a GSA Schedule
  - Type: External
  - [See if the Schedule is a good fit for your business first.](https://www.gsa.gov/buy-through-us/purchasing-programs/multiple-award-schedule/help-with-mas-contracts-to-sell-to-government/team-up-with-other-mas-contractors)
- On Page: https://www.idmanagement.gov/gsapkissp/
  - Fixed link in section: Step 5 -Apply to GSA MAS and Get Listed as an Identity Trusted Service
  - Under: Acquisition Procedural Guidance:
  - Updated Link text to match URL
  - Type: External
  - [https://www.gsa.gov/buy-through-us/purchasing-programs/multiple-award-schedule/help-with-mas-contracts-to-sell-to-government/roadmap-to-get-a-mas-contract](https://www.gsa.gov/buy-through-us/purchasing-programs/multiple-award-schedule/help-with-mas-contracts-to-sell-to-government/roadmap-to-get-a-mas-contract)
- On Page: https://www.idmanagement.gov/university/pacs/
  - Fixed link in section: What Other GSA Resources Can Help Us?
  - Type: External
  - [GSA Schedules - General Information](https://www.gsa.gov/buy-through-us/purchasing-programs/multiple-award-schedule/help-with-mas-buying)
- On Page: https://www.idmanagement.gov/implement/announcements/PT-TLS-CP/
  - Announcement Title: Public Trust PKI Certificate Policy Update
  - Fixed link in section: US Federal Public Trust TLS Certificate Policy
  - Type: Internal
  - [US Federal Public Trust TLS Certificate Policy](https://www.idmanagement.gov/docs/archived/us-federal-public-trust-tls-cp-v0-1.pdf)
- On Search Page: https://www.idmanagement.gov/search/?utf8=%E2%9C%93&affiliate=idmprod&query=
  - Updated links on search page to appropriate  content:
  - Type: Internal
  - Note:  👀 noticed other links on page needing to be updated, in addition to what SiteImprove found, updated links.
  - [FPKI Compliance](https://www.idmanagement.gov/fpki/#audit-information-for-the-fpki-management-authority)
  - [Trust Services](https://www.idmanagement.gov/trust-services/)
  - [Approved Products List](https://www.idmanagement.gov/fips201/)
  - [Buy FICAM Products and Services](https://www.idmanagement.gov/fips201/#how-to-purchase)
  - [Sell FICAM Products and Services](https://www.idmanagement.gov/acquisition-professionals/#products)
  - [Manage ICAM Programs](https://www.idmanagement.gov/university/pm/)
  - [Build ICAM Systems](https://www.idmanagement.gov/playbooks/)

@SuGhadiali 
@TheInfinityBeyonder 
@KonradMiles 
@JBPayne007
